File: program/find-largest-and-smallest-numbers-in-an-array/find_largest_and_smallest_numbers_in_an_array.php
<?php

function find_largest_and_smallest_numbers_in_an_array($numbers)
{
	sort($numbers);
	$largestNumber = $numbers[count($numbers) - 1];
	$smallestNumber = $numbers[0];
	return "$largestNumber, $smallestNumber";
}

$inputs = [1, 2, 3, 4, 5];
echo find_largest_and_smallest_numbers_in_an_array($inputs);

File: program/program/find-largest-and-smallest-numbers-in-an-array/FindLargestAndSmallestNumbersInAnArray.kt
fun findLargestAndSmallestNumbers(numbers: Array<Int>): String {
	numbers.sort()
	val largestNumber = numbers[numbers.size - 1]
	val smallestNumber = numbers[0]
	return "${largestNumber}, ${smallestNumber}"
}

fun main() {
	val inputs = arrayOf<Int>(1, 2, 3, 4, 5)
	println(findLargestAndSmallestNumbers(inputs))
}
